sources/org.osbuild.ostree

```diff
@@ -97,29 +92,33 @@ def download(items, cache):
         ostree("remote", "add",
                uid, url,
                *verify_args,
-               repo=repo_cache)
+               repo=self.repo_cache)
```

Comment:
If you do a `repo_cache = self.repo_cache` at the top of the function you don't have to change every instance of this.